libminifi/src/sitetosite/RawSocketProtocol.cpp

```diff
@@ -519,7 +519,7 @@ bool RawSiteToSiteClient::bootstrap() {
   tearDown();
 
   if (establish() && handShake() && negotiateCodec()) {
-    logger_->log_info("Site2Site Ready For data transaction");
+    logger_->log_trace("Site2Site Ready For data transaction");
```

Comment:
If you were to set the log level to debug one might expect there to be something here, as you would need to select trace and inevitably trace the entire flow of execution vice getting debug information.